src/main/java/com/github/susom/dbgoodies/etl/BigQueryWriter.java

```diff
@@ -557,51 +563,51 @@ public void run() {
     private BigQueryWriterBuilder() {
     }
 
-    public static BigQueryWriterBuilder aBigQueryWriter() {
+    static BigQueryWriterBuilder aBigQueryWriter() {
```

Comment:
This strikes me as an odd way to initialize a builder (`BigQueryWriter.BigQueryWriterBuilder.aBigQueryWriter()`). Wouldn't a more conventional way to be to name the class `Builder` with a public constructor and static method in the outer class? I.e. `new BigQueryWriter.Builder()` and `BigQueryWriter.builder()` as creation options.

Reply:
the code is generated by Builder Generator, https://github.com/mjedynak/builder-generator-idea-plugin. Maybe I can fork the plugin and make the code style compliant with our standard.

Reply:
Renaming the class and moving the static method is probably easier than rewriting an IntelliJ plugin... :)

Reply:
true. I manually changed the builder code this time.  I will look for another builder plugin in the future.